sormas-ui/src/main/java/de/symeda/sormas/ui/caze/CaseFilterForm.java

```diff
@@ -144,6 +146,14 @@ public void addMoreFilters(CustomLayout moreFiltersContainer) {
 			officerField.addItems(FacadeProvider.getUserFacade().getUsersByRegionAndRoles(user.getRegion(), UserRole.SURVEILLANCE_OFFICER));
 		}
 
+		Field<?> followUpUntilTo = addField(
```

Comment:
Don't add this filter if follow-up is not enabled for the server

Reply:
OK